ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/BackupReplayFile.java

```diff
@@ -30,21 +31,84 @@
 import java.io.OutputStreamWriter;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.PosixFilePermissions;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
 
 /**
  * A file that is used by the backup service to replay command_topic commands.
  */
-public class BackupReplayFile implements Closeable {
-  private static final ObjectMapper MAPPER = PlanJsonMapper.INSTANCE.get();
+public final class BackupReplayFile implements Closeable {
+  enum Versions {
+    NO_VERSION, V1
+  }
+
+  // Current version of the Backup file
+  private static final String BACKUP_VERSION_HEADER = "BACKUP_VERSION";
+  private static final Versions DEFAULT_BACKUP_VERSION = Versions.V1;
+
+  // Include.ALWAYS is necessary to serialize nulls and empty values. This is required to allow
+  // the CommandTopicBackupImpl to compare against all properties from a record read from
+  // the command topic, which come with nulls and empty values.
+  private static final ObjectMapper MAPPER = PlanJsonMapper.INSTANCE.get()
+      .setSerializationInclusion(JsonInclude.Include.ALWAYS);
```

Comment:
I think this will set it on the underlying instance. Instead we should create a new instance and set the serialization on the new instance (we'll need to change `PlanJsonMapper` to expose a newInsatcne() call)